Analyzer/include/Config.h

```diff
@@ -42,6 +42,7 @@ class Config
         const auto& filetag               = tr.getVar<std::string>("filetag");
         const auto& TopTaggerCfg          = tr.getVar<std::string>("TopTaggerCfg");       
  
+        std::string suffix = "";
```

Comment:
I don't like this. Why are you using it? If you want to do this put this below with the other hard coded variables.